### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "mariadb-server, php-pear, php8.2-ldap, php8.1-mysql, php8.1-cli, php8.1-intl, php8.1-zip, php8.1-gd, php8.1-mbstring, php8.1-dom, php8.1-curl"
```
Comment (alexAubin) [inline manifest.toml:68]:
```suggestion
    packages = "mariadb-server, php-pear, php8.1-ldap, php8.1-mysql, php8.1-cli, php8.1-intl, php8.1-zip, php8.1-gd, php8.1-mbstring, php8.1-dom, php8.1-curl"
```

8.2 -> 8.1 for consistency

File: scripts/install [hunk 2]
```diff
@@ -130,24 +66,23 @@ ynh_install_composer
...
 # INSTALL ADDITIONAL PLUGINS
 #=================================================
 ynh_script_progression --message="Installing additional plugins..." --weight=60
 
 # Create logs and temp directories
-mkdir -p "$final_path/"{logs,temp}
+mkdir -p "$install_dir/"{logs,temp}
 
 # Install net_LDAP
 ynh_composer_exec --commands="require kolab/net_ldap3"
```
Comment (alexAubin) [inline scripts/install:88]:
```suggestion
export COMPOSER_ALLOW_SUPERUSER=1
ynh_composer_exec --commands="require kolab/net_ldap3"
```

**Comment by alexAubin** [comment]
```
25160 INFO DEBUG - + ./bin/update.sh '--version=?' -y
25160 INFO WARNING - Composer detected issues in your platform:
25160 INFO DEBUG -
25160 INFO WARNING - Your Composer dependencies require a PHP version ">= 8.1.0". You are running 7.4.33.
25160 INFO DEBUG -
25160 INFO WARNING - PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 7.4.33. in /var/www/roundcube/vendor/composer/platform_check.php on line 24
```

sounds related to the stupid issues with bin/update.sh we're seeing on the testing PR

**Comment by Salamandar** [comment]
Rebased.

**Comment by Salamandar** [comment]
!testme

**Pull request merged**